docs/modules/transform-sync/how-it-works.md

```diff
@@ -46,8 +37,8 @@ component TransformInternal {
 There are a few reasons why you might want different `Transform` and `Position` components.
 
 1. **Separation of responsibilies.** The `Improbable.Position` component is the load balancer's representation of location and the `TransformInternal` component is the workers' representation of location. This allows you to abstract the concept of location from the load balancer. A gameplay programmer would only need to be aware of the `TransformInternal` component without having to know about the load balancer at all.
-2. **Save bandwidth.** The `Position` component represents location with 3 doubles. The `TransformInternal` component represents location as 3 floats, which is the Unity native representation. This means that, ignoring rotation, a position update is twice as large as a transform update. If the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate then you have a net bandwidth saving over updating just the `Position` component at a high frequency!
-3. **Atomicity.** The `Transform` component contains more than just the location. If the other fields were on a separate component, you lose the guarantee that all fields are updated atomically.<br/><br/>Two updates sent in the same frame from one worker are _not_ guaranteed to be received on the same frame in another.
+2. **Save bandwidth.** The `Position` component represents location with 3 doubles. The `TransformInternal` component represents location as 3 fixed point value. This means that, ignoring rotation and velocity, a position update is _at least_ twice as large as a location update. If the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate, then you have a net bandwidth saving compared to updating just the `Position` component at a high frequency!
```

Comment:
Given we have compressed them - call it out here?

Reply:
restructured this section